draft-privacypass-rate-limit-tokens.md

```diff
@@ -1460,14 +1441,14 @@ identity (as known to the Attester) to the Origin, especially if repeated over m
 
 ## Token Key Management
 
-Issuers SHOULD generate a new (Token Key, Issuer Origin Secret) regularly, and
```

Comment:
Should this pick up a line about why token key reuse across origins would be a bad idea?

Reply:
Yep, added.